package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "prx-ng-serve",
-  "version": "0.0.4",
-  "description": "Express server for PRX angular 2 applications",
+  "version": "0.1.0",
```

Comment:
I'm feeling like this should be `1.0.0`.  Since there is no upgrade-path (without also upgrading angular).  But I can be convinced!

Reply:
Agreed.